INSTALL.md

```diff
@@ -46,7 +46,7 @@ CMake option `CMAKE_PREFIX_PATH` and point it to the folder of your installation
 For example:
 
 ```bash
-    cmake -DCMAKE_PREFIX_PATH=$HOME/Qt/5.11.2/gcc_64 ..
+    cmake -DCMAKE_PREFIX_PATH=$HOME/Qt/5.15.4/gcc_64 ..
 ```
```

Comment:
Maybe this should just use `$HOME/Qt/6.6.0/gcc_64` as an example, instead? (Along with the envvar version below.)

Doesn't hurt to _encourage_ users to migrate to Qt6, after all.

Reply:
good idea. I'll do that